core/src/main/java/hudson/security/csrf/CrumbIssuerDescriptor.java

```diff
@@ -64,7 +64,7 @@ public String getCrumbRequestField() {
      */
     public void setCrumbRequestField(String requestField) {
         if (Util.fixEmptyAndTrim(requestField) == null) {
-            crumbRequestField = ".crumb";
+            crumbRequestField = CrumbIssuer.DEFAULT_CRUMB_NAME;
```

Comment:
Never mind, I got it backwards. It is actually `DefaultCrumbIssuer.DescriptorImpl.<init>` which is doing something redundant. It could just pass `null` normally to the super constructor:

```java
super(CRUMB_SALT.get(), System.getProperty("hudson.security.csrf.requestfield"));
```

so that (except for tests) the only place `DEFAULT_CRUMB_NAME` need be used is here, in `CrumbIssuerDescriptor.setCrumbRequestField`, so the constant could sensibly be moved here from `CrumbIssuer`.